core/renderers/geras/renderer.ts

```diff
@@ -51,7 +51,7 @@ export class Renderer extends BaseRenderer {
    *
    * @internal
    */
-  override init(theme: Theme, opt_rendererOverrides: AnyDuringMigration) {
+  override init(theme: Theme, opt_rendererOverrides?: object) {
```

Comment:
This is typed as `{[rendererConstant: string]: any}` in `block_rendering.ts`

Reply:
Updated accordingly.